pkgs/dkg/drand.go

```diff
@@ -22,33 +18,46 @@ import (
 	eth_crypto "github.com/ethereum/go-ethereum/crypto"
 	"github.com/herumi/bls-eth-go-binary/bls"
 	"github.com/pkg/errors"
-	"github.com/spf13/viper"
 	"go.uber.org/zap"
+
+	"github.com/bloxapp/ssv-dkg/pkgs/board"
+	"github.com/bloxapp/ssv-dkg/pkgs/crypto"
+	"github.com/bloxapp/ssv-dkg/pkgs/utils"
+	"github.com/bloxapp/ssv-dkg/pkgs/wire"
 )
 
+var OutputPath string
+var StoreShare bool
+
 const (
 	// MaxEffectiveBalanceInGwei is the max effective balance
 	MaxEffectiveBalanceInGwei phase0.Gwei = 32000000000
-
 	// BLSWithdrawalPrefixByte is the BLS withdrawal prefix
 	BLSWithdrawalPrefixByte = byte(0)
 )
 
 // IsSupportedDepositNetwork returns true if the given network is supported
 var IsSupportedDepositNetwork = func(network eth2_key_manager_core.Network) bool {
```

Comment:
I'm pretty sure I removed this in my holesky_support, this function exist in `eth2_key_manager` so its better to use it from there than rewriting.

Reply:
I merged to here `holesky_support`. This is probably a leftover. Ill remove